.github/workflows/validate_and_release.yml

```diff
@@ -36,6 +36,33 @@ jobs:
         python scripts/jsonParser.py
         reproschema validate examples
 
+  generate_other_formats:
```

Comment:
note that this will generate the files on every PR but that they will no be added to the repo or the release

files generated on a given job are not passed the following ones in github action, so maybe better to have the file generation happen in the release job

Reply:
yes, I didn't get the structure at the beginning, just moved

Reply:
will be testing `|| true`